src/kubelet/kubelet_test.go

```diff
@@ -26,19 +28,11 @@ import (
 )
 
 func TestScraper(t *testing.T) {
-
-	commonMetricsToExclude := []string{"net.rxBytesPerSecond", "net.txBytesPerSecond", "net.errorsPerSecond"}
-	nodeMetricsToExclude := append(commonMetricsToExclude, "allocatableCpuCoresUtilization", "allocatableMemoryUtilization")
-
 	// Create an asserter with the settings that are shared for all test scenarios.
 	asserter := asserter.New().
+		Silently().
```

Comment:
I am happy to see that, but I fear it is a leftover. Sure you do not want to keep it?

Reply:
Hahah it is 100% a leftover :P